### PR TITLE
Add Quplexity to README.md

### PR DESCRIPTION
**Pull request by MrGilli**
Added the Quplexity modular library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [XACC](https://github.com/ORNL-QCI/xacc) - Extreme-scale programming model for quantum acceleration within high-performance computing ([arXiv paper](https://arxiv.org/abs/1710.01794)).
 - [qpp](https://github.com/softwareQinc/qpp) - Quantum++ is a modern C++ general purpose quantum computing library, composed solely of template header files.
 
+**Assembly**
+- [Quplexity](https://github.com/MrGilli/Quplexity) - Modular library/toolkit for QC sims, including key mathematical functions and logic its written in ARM(64) and x86 Assembly, which results in improved performance and "weight" benefits.
```
Comment (dlyongemallo) [inline README.md:50]: > logic its written

Is the "its" a typo?

Comment (MrGilli) [reply README.md:50]: Sorry my apologies, I will fix It and create another pull request.

Comment (MrGilli) [reply README.md]: Ok, the mistake is fixed in the new pull request, sorry.

File: README.md [hunk 2]
```diff
@@ -46,6 +46,9 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [XACC](https://github.com/ORNL-QCI/xacc) - Extreme-scale programming model for quantum acceleration within high-performance computing ([arXiv paper](https://arxiv.org/abs/1710.01794)).
 - [qpp](https://github.com/softwareQinc/qpp) - Quantum++ is a modern C++ general purpose quantum computing library, composed solely of template header files.
 
+**Assembly**
```
Comment (dlyongemallo) [inline README.md:49]: I'm not sure this belongs under "full-stack libraries". It seems to provide simulation routines but there doesn't appear to be any front end? Maybe it belongs under "Quantum simulators" instead? Also, the languages are sorted alphabetically, so "Assembly" should go before "C++".

Comment (MrGilli) [reply README.md:49]: Ok, I'll add it under Quantum Simulators instead, with the "Assembly" heading before "C".

Comment (MrGilli) [reply README.md]: I've submitted the updated pull request.

**Pull request merged**